### DISCUSSION TITLE
Split screen usage

### DISCUSSION THREAD

**Discussion by wayfarren** [kylecorry31/Trail-Sense#1831] [New Feature Ideas]
I think it would be awesome if I could open trail sense up in a multi window view. For example, google maps on the left, and the trail sense compass on the right. It seems that this was previously an issue with trail sense, being that it doesn't display properly, so it was turned off.

Perhaps the user can select what view they want to have when opening split-screen?

As an example, i could start the split screen view and open trail sense, and trail sense asks me what i want displayed.

If detecting split screen isn't possible like this, perhaps there could be a split screen button for every tool which simplifies and optimizes the UI, making it optimal for split screen?

**Comment by kylecorry31** [comment]
Hi @HenriHawk42 , thank you for the suggestion, I'll create an issue to investigate readding split screen though I'm not sure how likely it will be. The main reason I removed it is because I didn't design Trail Sense with a very small screen in mind, and doing so may require a large rework of tools which isn't feasible at this point. At the time I removed it, almost all phones had stacked split screen (ex. half the phone height), but now I know there are devices with wide screens (ex. foldables) so it may actually work fine on them (as you mentioned the app can be side by side instead).

Here's the issue to track it https://github.com/kylecorry31/Trail-Sense/issues/1832

**Reply by wayfarren** [reply]
Thanks! For the sake of accuracy, I am not using a foldable device, just a normal phone horizontally. I completely understand if this kind of feature isn't feasible. Thanks for the quick reply!
